### ISSUE TITLE
Nodejs support 

### ISSUE THREAD

**Issue by temaivanoff** [paullouisageneau/datachannel-wasm#43]
Hi, this wasm lib work to Nodejs ? Nodejs not support native webrtc. I can extend functionality with this library for nodejs ?

**Comment by paullouisageneau** [comment]
No, this is just a wrapper that exposes the existing WebRTC JS API to C++ wasm programs, it needs the WebRTC implementation to be there already.

You might be interested in [node-datachannel](https://github.com/murat-dogan/node-datachannel) which adds WebRTC support to node.js.

**Comment by jimmywarting** [comment]
but NodeJS has `globalThis.WebAssembly`. dose it not work?

**Comment by paullouisageneau** [comment]
> but NodeJS has `globalThis.WebAssembly`. dose it not work?

What do you mean? This is a C++ wrapper for the existing WebRTC JavaScript API, it does not provide the implementation by itself.

**Comment by paullouisageneau** [comment]
> but NodeJS has `globalThis.WebAssembly`. dose it not work?

What do you mean? This is a C++ wrapper for the existing WebRTC JavaScript API (which Node.js does not support natively), it does not provide the implementation by itself.
